RELEASES.md

```diff
@@ -106,21 +106,26 @@ convert to a regular _Active_ release with stricter backport criteria.
-| [2.0](https://github.com/containerd/containerd/milestone/35)         | Next          | TBD                | TBD                                              |
-
+| Release                                                              | Status        | Start              | End of Life                                             |
+| ---------                                                            | ------------- | ------------------ | -------------------                                     |
+| [0.0](https://github.com/containerd/containerd/releases/tag/0.0.5)   | End of Life   | Dec 4, 2015        | -                                                       |
+| [0.1](https://github.com/containerd/containerd/releases/tag/v0.1.0)  | End of Life   | Mar 21, 2016       | -                                                       |
+| [0.2](https://github.com/containerd/containerd/tree/v0.2.x)          | End of Life   | Apr 21, 2016       | December 5, 2017                                        |
+| [1.0](https://github.com/containerd/containerd/releases/tag/v1.0.3)  | End of Life   | December 5, 2017   | December 5, 2018                                        |
+| [1.1](https://github.com/containerd/containerd/releases/tag/v1.1.8)  | End of Life   | April 23, 2018     | October 23, 2019                                        |
+| [1.2](https://github.com/containerd/containerd/releases/tag/v1.2.13) | End of Life   | October 24, 2018   | October 15, 2020                                        |
+| [1.3](https://github.com/containerd/containerd/releases/tag/v1.3.10) | End of Life   | September 26, 2019 | March 4, 2021                                           |
+| [1.4](https://github.com/containerd/containerd/releases/tag/v1.4.13) | End of Life   | August 17, 2020    | March 3, 2022                                           |
+| [1.5](https://github.com/containerd/containerd/releases/tag/v1.5.18) | End of Life   | May 3, 2021        | February 28, 2023                                       |
+| [1.6](https://github.com/containerd/containerd/releases/tag/v1.6.23) | LTS           | February 15, 2022  | max(February 15, 2025 or next LTS + 6 months)           |
+| [1.7](https://github.com/containerd/containerd/releases/tag/v1.7.3)  | Active        | March 10, 2023     | active(release of 2.0 + 6 months), extended(EOL of 1.6) |
+| [2.0](https://github.com/containerd/containerd/milestone/35)         | Next          | TBD                | TBD                                                     |
+
+** Note: containerd v1.7 will end of life at the same time as v1.6 LTS. Due to
+Minimal Version Selection used by Go modules, 1.7 must be supported until EOL of
+all 1.x releases. Once 1.7 is in extended support, it will continue to accept
+security patches in addition to client changes relevant for package importers
```

Comment:
q: Does this mean that, after release of 2.0 + 6 months, patches will be directly cherry picked to 1.6 instead of the current 1.7 and 1.6 process that we follow.

Reply:
Some patches should skip 1.7 but up to maintainer discretion. If a change is not security related and only has an effect on daemons (such as a shim fix), then it wouldn't make sense to backport into 1.7 at that point. The expectation is that users who are reluctant to upgrade would deploy 1.6 and users who want latest features will deploy 2.0. This would be the recommendation when 1.7 issues are opened, but whether the issue is affects importers or is covered by extended support is always open for debate.